src/app/modules/onboarding/module.nim

```diff
@@ -5,25 +5,59 @@ import view, controller
 
 import app/global/global_singleton
 import app/core/eventemitter
+import app_service/service/general/service as general_service
+import app_service/service/accounts/service as accounts_service
+import app_service/service/devices/service as devices_service
+import app_service/service/keycardV2/service as keycard_serviceV2
 
 export io_interface
 
 logScope:
   topics = "onboarding-module"
 
+type PrimaryFlow* {.pure} = enum
+  Unknown = 0,
+  CreateProfile,
+  Login
```

Comment:
Soo, do we need `PrimaryFlow`, if all of the values in `SecondaryFlow` have prefix indicating the primary flow? Looks absolutely redundant 🤔

Reply:
Seems like you're right. The QML only provides the secondary flow, which is sufficient anyway:

```
!!! ONBOARDING FINISHED; flow: 1 ; data: {\"password\":\"1234567890\",\"keycardPin\":\"\",\"seedphrase\":\"\",\"enableBiometrics\":false}
```

I'll get rid of the primary flow complexity